runtime/src/accounts_db.rs

```diff
@@ -7033,24 +7033,26 @@ impl AccountsDb {
                 {
                     let mut load_from_cache = true;
                     let mut hasher = std::collections::hash_map::DefaultHasher::new();
+                    bin_range.start.hash(&mut hasher);
+                    bin_range.end.hash(&mut hasher);
```

Comment:
It will change the hash. It wasn't new information to hash the start and end range for each slot in the range.

Invalidating the files on disk currently is perfectly fine. We expect all cache files to be invalid when we restart anyway.